pywavefront/parser.py

```diff
@@ -57,14 +61,6 @@ def parse(self, line, dir):
 
         line_type = values[0]
         args = values[1:]
-        i = 0
-        for arg in args:
-            if dir != '' and ('mtllib' in line or 'map_Kd' in line):
-                args[i] = dir + '/' + arg
-            else:
-                args[i] = arg
-            i += 1
-
```

Comment:
This was a good catch.

Reply:
Blame the profiler. It was creaming at those lines 😆